mmengine/optim/optimizer/optimizer_wrapper.py

```diff
@@ -276,7 +301,16 @@ def param_groups(self) -> List[dict]:
         Returns:
              dict: the ``param_groups`` of :attr:`optimizer`.
         """
-        return self.optimizer.param_groups
+
+        # If ``self.base_param_settings`` is not None, the ``param_groups``
+        # of :attr:`optimizer` will be appended to
+        # ``self.base_param_settings``. Otherwise, the ``param_groups``
+        # of :attr:`optimizer` will be returned.
+        if self.base_param_settings is not None:
+            return list(
+                chain(self.optimizer.param_groups, [self.base_param_settings]))
```

Comment:
Can we directly return `self.optimizer.param_groups + [self.base_param_settings]`?

Reply:
good idea.